Makefile

```diff
@@ -1,3 +1,4 @@
+DOCKER_COMPOSE_BETA := docker-compose -p $$(git rev-parse --abbrev-ref HEAD)
```

Comment:
Doesn't this get the branch name? I think we instead want some thing like `git rev-parse HEAD` or `git rev-parse --short HEAD` for getting the commit hash.